src/App.js

```diff
@@ -56,13 +60,60 @@ class App extends Component {
       })
   }
 
+  checkKey(e) {
+
+      if (e.repeat == false){
+
+        let theKeyPressed = null;
+        let {ctrl, shift} = this.state.hotkeys;
+
+        if(window.event) {
+          theKeyPressed = e.keyCode;
+        }
+
+        if(e.which) {
+          theKeyPressed = e.which;
+        }
+
+      switch(theKeyPressed) {
```

Comment:
yep there was a bigger case thats why i used a switch nw its simplified so ill remove it :)